plugins/GSdx/GSDump.cpp

```diff
@@ -51,7 +51,7 @@ void GSDumpBase::Transfer(int index, const uint8* mem, size_t size)
 		return;
 
 	AppendRawData(0);
-	AppendRawData(index);
+	AppendRawData(static_cast<uint8>(index));
```

Comment:
Dunno, if it wouldn't be better to use `int` in the interface. And add an assert if the value is > 255. But I'm fine with any way.